content/en/providers/operations-manuals/howto21_adding_service_endpoint_into_gocdb/_index.md

```diff
@@ -0,0 +1,175 @@
+---
+title: HOWTO21
+weight: 135
+type: "docs"
+description: "How to add URL service endpoint information into GOC-DB"
+---
+
+For monitoring purposes, each service endpoints registered into GOC-DB
+with the flags *production* and *monitored* should include the endpoint
+URL information in order to be contacted by the relative nagios probe.
+
+**Important: currently these steps are required only for the sites under
+certification. For sites in production, EGI Monitoring is retrieving
+from the Top-BDII these information**
+
+The information needed for service type are:
+
+  - CREAM-CE: the queue URL where the ops VO is enabled. This is the
+    value of the attribute GlueCEUniqueID published in the BDII (e.g.
+    *cream.egi.eu:8443/cream-pbs-ops*).
+  - SRM: the value of the attribute GlueServiceEndpoint published in the
+    BDII (e.g. *httpg://se.egi.eu:8444/srm/managerv2*)
+  - Other service types: the value of the attribute GlueServiceEndpoint
+    published in the BDII
+
+It is also possible registering additional endpoints for each service:
+by setting also the "Monitored" flag, ARGO will monitor them.
+
+For having more information about managing the Service endpoints in
+GOC-DB, please consult the relative
+[documentation](https://wiki.egi.eu/wiki/GOCDB/Input_System_User_Documentation#Service_Endpoints).
+
+## Get the information
+
+For retrieving the queue URL from the BDII, you can simply use the
+command *lcg-infosites* executed from an UI. Be sure to query a
+production Top-BDII: you can either set the one provided by your
+Operations Centre or choose one from
+<https://goc.egi.eu/portal/index.php?Page_Type=Service_Group&id=1205>
+
+For example:
+
+`$ export LCG_GFAL_INFOSYS=egee-bdii.cnaf.infn.it:2170`
+
+`$ lcg-infosites --vo ops ce | grep nikhef`
+`  5680      15          0            0       0 dissel.nikhef.nl:2119/jobmanager-pbs-infra`
+`  5680      17          1            1       0 gazon.nikhef.nl:8443/cream-pbs-infra`
+`  5680      15          0            0       0 juk.nikhef.nl:8443/cream-pbs-infra`
+`  5680      15          0            0       0 klomp.nikhef.nl:8443/cream-pbs-infra`
+`  5680      16          0            0       0 stremsel.nikhef.nl:8443/cream-pbs-infra`
+
```

Comment:
better a code block

Reply:
(All this was an automatic import for whose I didn't reviewed the syntax)